rllib/optimizers/async_samples_optimizer.py

```diff
@@ -79,7 +79,7 @@ def __init__(self,
 
         # Stats
         self._optimizer_step_timer = TimerStat()
-        self._stats_start_time = time.time()
+        self._stats_start_time = time.perf_counter()
```

Comment:
I'm not sure about this; is `self. _stats_start_time` used anywhere else, where the actual time is needed, as opposed to the time taken?